spec/mailers/candidate_mailer_offers_and_rejections_spec.rb

```diff
@@ -232,7 +232,7 @@
   describe '.reinstated_offer' do
     let(:email) { mailer.reinstated_offer(application_choices.first) }
     let(:application_choices) { [application_choice] }
-    let(:application_choice) { build_stubbed(:application_choice, :with_deferred_offer, course_option: other_option, offer_deferred_at: Time.zone.local(2019, 10, 3)) }
+    let(:application_choice) { build_stubbed(:application_choice, :with_deferred_offer, course_option: other_option, current_option: other_option, offer_deferred_at: Time.zone.local(2019, 10, 3)) }
```

Comment:
Why doesn't the `after(:build)` hook catch this and assign `course_option` to `current_option`?

Reply:
I think `after(:build)` is used if data is set up via `.build` or `.create`, but `.build_stubbed` triggers `after(:stub)` without triggering `after(:build)`.